front/app/containers/ProjectsShowPage/index.tsx

```diff
@@ -87,12 +95,32 @@ const ProjectsShowPage = memo<Props>(({ project, scrollToEventId }) => {
     ? project.attributes.process_type
     : undefined;
 
+  const smallerThanMinTablet = useBreakpoint('tablet');
+  const { formatMessage } = useIntl();
   const [showModal, setShowModal] = useState<boolean>(false);
   const [phaseIdUrl, setPhaseIdUrl] = useState<string | null>(null);
   const locale = useLocale();
   const appConfig = useAppConfiguration();
   const phases = usePhases(projectId);
 
+  const { events } = useEvents({
```

Comment:
This useEvents here isn't really needed, but I have a remaining question for Ben about the behaviour. So just leaving it in for now.

Reply:
Were you able to find out what you needed from Ben here?

Reply:
Ah yes! I did and made the changes :) I'll remove this variable :+1: